atomic

```diff
@@ -514,6 +518,18 @@ def create_parser(help_text):
                      help=_("Only display container IDs"))
     add_opt(pss)
 
+    # atomic sign
+    signer = ATOMIC_CONFIG.get('default_signer', None)
+    sig_store = ATOMIC_CONFIG.get('default-sigstore-path', "/var/lib/atomic/containers")
+    signp = subparser.add_parser("sign", help="Sign an image",
```

Comment:
Can you move all of this handling to sign.py under a function call cli, then only change to atomic is
sign.cli(subparser)

Just like we have done for all of the other CLI Commands.